numpy/core/src/multiarray/dragon4.c

```diff
@@ -1442,8 +1454,8 @@ typedef union FloatUnion16
 } FloatUnion16;
 
 npy_bool   IsNegative_F16(FloatUnion16 *v) { return (v->integer >> 15) != 0; }
-npy_uint32 GetExponent_F16(FloatUnion16 *v) { return (v->integer >> 10) & 0x1F;}
-npy_uint32 GetMantissa_F16(FloatUnion16 *v) { return v->integer & 0x3FF; }
+npy_uint32 GetExponent_F16(FloatUnion16 *v) { return (v->integer >> 10) & bitmask_u32(5);}
+npy_uint32 GetMantissa_F16(FloatUnion16 *v) { return v->integer & bitmask_u32(10); }
```

Comment:
These lines are my own changes, since #11167 removes them completely.

Should be straightforward to discard these (and all other changes) when rebasing